I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/SCRIPT/characters.rpy", line 117, in script
    init python:
  File "game/SCRIPT/characters.rpy", line 117, in script
    init python:
  File "game/SCRIPT/characters.rpy", line 124, in <module>
    p_team= wrestling_team(members=[mistico, wagner, yukiko])
  File "game/SCRIPT/characters.rpy", line 13, in __init__
    sel.moves_act=[]
NameError: global name 'sel' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 326, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 515, in main
    renpy.game.context().run(node)
  File "game/SCRIPT/characters.rpy", line 117, in script
    init python:
  File "game/SCRIPT/characters.rpy", line 117, in script
    init python:
  File "renpy/ast.py", line 913, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2111, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/SCRIPT/characters.rpy", line 124, in <module>
    p_team= wrestling_team(members=[mistico, wagner, yukiko])
  File "game/SCRIPT/characters.rpy", line 13, in __init__
    sel.moves_act=[]
NameError: global name 'sel' is not defined

